src/Scene.test.js

```diff
@@ -227,10 +231,55 @@ describe('When the robot character renders, transform should apply', () => {
     test('When xPos = 0, yPos = 90, directionDegrees = 0', () => {
         expect.assertions(1);
         const sceneWrapper = createMountScene({
-            characterState: new CharacterState(0, 90, 0)
+            characterState: new CharacterState(0, 90, 0, [])
         });
         const robotCharacter = findRobotCharacter(sceneWrapper);
         expect(robotCharacter.get(0).props.transform)
             .toBe('translate(0 90) rotate(-90 0 0)');
     });
 });
+
+describe('When the robot character moves, it will draw its moving path on the scene', () => {
```

Comment:
Please rename this test to "When the Character has a path, it is drawn on the Scene"